exonum-java-binding/testkit/src/test/java/com/exonum/binding/testkit/TestKitTest.java

```diff
@@ -467,52 +443,6 @@ void createBlockWithTransactions(TestKit testKit) {
         view, block, message, message2));
   }
 
-  @Test
-  void nodeSubmittedTransactionsArePlacedInPool(TestKit testKit) {
-    TestService service = testKit.getService(SERVICE_NAME, TestService.class);
-
-    TransactionMessage message = constructTestTransactionMessage("Test message", testKit);
-    RawTransaction rawTransaction = RawTransaction.fromMessage(message);
-    service.getNode().submitTransaction(rawTransaction);
-
-    List<TransactionMessage> transactionsInPool =
-        testKit.findTransactionsInPool(tx -> tx.getServiceId() == SERVICE_ID);
-    assertThat(transactionsInPool).isEqualTo(ImmutableList.of(message));
-  }
-
-  @Test
-  void getTransactionPool(TestKit testKit) {
-    TestService service = testKit.getService(SERVICE_NAME, TestService.class);
-
-    TransactionMessage message = constructTestTransactionMessage("Test message", testKit);
-    RawTransaction rawTransaction = RawTransaction.fromMessage(message);
-    TransactionMessage message2 = constructTestTransactionMessage("Test message 2", testKit);
-    RawTransaction rawTransaction2 = RawTransaction.fromMessage(message2);
-
-    service.getNode().submitTransaction(rawTransaction);
-    service.getNode().submitTransaction(rawTransaction2);
-
-    List<TransactionMessage> transactionsInPool = testKit.getTransactionPool();
-    assertThat(transactionsInPool).containsExactlyInAnyOrder(message, message2);
-  }
-
-  @Test
-  void findTransactionsInPool(TestKit testKit) {
```

Comment:
`TestKit.findTransactionsInPool()` is tested in `afterCommitSubmitsTransaction` test.